### PR TITLE
fix preview bug

### LINKED ISSUE #1643: preview bug :  component moves up
**I'm submitting a ...**
- [X] bug report
- [ ] feature request

the component moves up when clicked on toggle preview.

### PR DESCRIPTION
**Pull request by YogeshSharma0201**
Fixes #1643

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

The component was moving up and down because of the main-container class, it specifies padding and margins in %, which changes dynamically when its width is changed.
I have also removed some duplicate lines in create-badge.scss.

### CODE REVIEW

**Comment by ParthS007** [comment]
@YogeshSharma0201 Please add an `After and Before` screenshot.

**Comment by YogeshSharma0201** [comment]
Before (The form is little higher than the preview component, before toggling preview button it was at the same level at which preview is now):
screenshot from 2018-10-19 17-21-33

After (Both components are at same level now):
screenshot from 2018-10-19 17-20-55

File: frontend/app/styles/partials/create-badge.scss [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   .column{
     color: black;
-    
+
     .create-badge {
       position: relative;
       padding-left: 6%;
       padding-right: 6%;
-      top: -35px;
+      top: 14px;
```
Comment (yashLadha) [inline frontend/app/styles/partials/create-badge.scss]: Can you please test the responsiveness?? That would be good.

**Comment by yashLadha** [comment]
Great :+1:

**Pull request merged**